<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>652</width>
    <height>505</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButtonRead">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>300</y>
      <width>101</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Read Card</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxEncryptWrite">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>200</y>
      <width>92</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Encrypt</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="plainTextEditKeyWrite">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>200</y>
      <width>361</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonWrite">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>150</y>
      <width>101</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Write Card</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="plainTextEditKeyRead">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>380</y>
      <width>361</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxDecryptRead">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>380</y>
      <width>92</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Decrypt</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelStatus">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>100</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Status</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelStatusMessage">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>100</y>
      <width>621</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBoxCom">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>40</y>
      <width>121</width>
      <height>28</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonCom">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>40</y>
      <width>101</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Refresh Ports</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditWrite">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>150</y>
      <width>381</width>
      <height>28</height>
     </rect>
    </property>
    <property name="maxLength">
     <number>32</number>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="plainTextEditRead">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>270</y>
      <width>381</width>
      <height>85</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonConnect">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>40</y>
      <width>90</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Open port</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>652</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
